padl/dumptools/symfinder.py

```diff
@@ -131,26 +132,24 @@ def deparse(self):
 
 
 def _fix_indent(source):
+    """Fix the indentation of functions that are wrongly indented.
+
+    This can happen with :func:`ast_utils.get_source_segment`.
+    """
     lines = source.lstrip().split('\n')
     res = []
     for line in lines:
         if line.startswith('@'):
             res.append(line.lstrip())
-        else:
+        elif line.lstrip().startswith('def ') or line.lstrip().startswith('class '):
```

Comment:
This `elif` and `else` could be combined into one as both end in `break`?

Reply:
:+1: